### PR TITLE
API Gateway gotcha Multi-Region

### PR DESCRIPTION
**Pull request by muhqu**
- API Gateway doesn't support Multi-Region.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1171,6 +1171,7 @@ API Gateway
 
 -	🔸API Gateway only supports encrypted (https) endpoints, and does not support unencrypted HTTP. (This is probably a good thing.)
 -	🔸API Gateway endpoints are always public, i.e. internet facing, and there is no mechanism to build private endpoints, e.g. for internal use on a [VPC](#vpcs-network-security-and-security-groups) but endpoints and their related resources can, optionally, [require authentication](http://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-to-api.html).
+-	API Gateway doesn't support Multi-Region. It is a service that is deployed in a single region but comes with a global endpoint that is served from Amazons Edge Locations (like a CloudFront Distribution). You can not have multiple API Gateways with the same hostname in different AWS Regions. ∙ [AWS Forum](https://forums.aws.amazon.com/thread.jspa?messageID=735342&#735342)
```
Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: API Gateway doesn't support Multi-Region. -> "API Gateway deployments aren't multi-region."

Comment (ThanosBaskous) [inline README.md]: Nit: I don't think "Edge Locations" should be capitalized -> "edge locations".

Comment (ThanosBaskous) [inline README.md]: `∙ [AWS Forum](https://forums.aws.amazon.com/thread.jspa?messageID=735342&#735342)`
I appreciate your trying to match the style here - I think that the "∙" is better as separator between multiple links.

Suggestion: `See this [AWS forum post](https://forums.aws.amazon.com/thread.jspa?messageID=735342&#735342) for more information.`

Comment (muhqu) [inline README.md]: 👍

Comment (muhqu) [inline README.md]: Ur right.

Comment (muhqu) [inline README.md]: That's not quite the same. I'm referring to "Multi-Region" as the name of a commonly known  design pattern for high availability. Will try to rephrase it again.

**Comment by muhqu** [comment]
@ThanosBaskous thx for the feedback. I did some changes, pls review.

File: README.md [hunk 2]
```diff
@@ -1171,6 +1171,7 @@ API Gateway
 
 -	🔸API Gateway only supports encrypted (https) endpoints, and does not support unencrypted HTTP. (This is probably a good thing.)
 -	🔸API Gateway endpoints are always public, i.e. internet facing, and there is no mechanism to build private endpoints, e.g. for internal use on a [VPC](#vpcs-network-security-and-security-groups) but endpoints and their related resources can, optionally, [require authentication](http://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-control-access-to-api.html).
+-	API-Gateway doesn't support multi-region deployments for high availability. It is a service that is deployed in a single region but comes with a global endpoint that is served from Amazons edge locations (like a CloudFront Distribution). You can not have multiple API Gateways with the same hostname in different AWS Regions and use Route 53 to distribute the traffic. See this [AWS forum post](https://forums.aws.amazon.com/thread.jspa?messageID=735342&#735342) for more information.
```
Comment (ThanosBaskous) [inline README.md]: Non-blocking nit (no need to fix before merge): Use unicode apostrophe (’).

Comment (ThanosBaskous) [inline README.md]: Non-blocking capitalization nit (no need to fix before merge): "AWS Regions" -> "AWS regions".

**Comment by jlevy** [comment]
👍 Thanks @muhqu !

**Pull request merged**